eng/Build.props

```diff
@@ -9,9 +9,9 @@
   </ItemDefinitionGroup>
 
   <ItemGroup>
-    <ProjectToBuild Include="$(RepoRoot)src/Redist/Microsoft.ML.DnnImageFeaturizer.ModelRedist/Microsoft.ML.DnnImageFeaturizer.ModelRedist.proj" />
-    <ProjectToBuild Include="$(RepoRoot)src/Native/Native.proj" />
-    <ProjectToBuild Include="$(RepoRoot)Microsoft.ML.sln" />
+    <ProjectToBuild Condition="'$(Build)' == true" Include="$(RepoRoot)src/Redist/Microsoft.ML.DnnImageFeaturizer.ModelRedist/Microsoft.ML.DnnImageFeaturizer.ModelRedist.proj" />
+    <ProjectToBuild Condition="'$(Build)' == true" Include="$(RepoRoot)src/Native/Native.proj" />
```

Comment:
Can you please add a note here about why the Condition flag is necessary?

Reply:
So its actually not needed. That was done to speed up compilation, but turns out the same thing happens if you dont have the condition and do /p:build=false. I will just go ahead and remove them.

Reply:
Where is /p:build=false specified?

---
In reply to: [526401865](https://github.com/dotnet/machinelearning/pull/5496#discussion_r526401865) [](ancestors = 526401865)

Reply:
And it looks like you added the comment but removed the condition. Am I reading that correctly?

---
In reply to: [526402860](https://github.com/dotnet/machinelearning/pull/5496#discussion_r526402860) [](ancestors = 526402860,526401865)

Reply:
I did add the comment, my bad. Its removed now. /p:build=false is specified in the job-template.yml file whenver we don't want to build. Otherwise the native code is restored each time, adding several minutes to each step.